packages/ckeditor5-dev-release-tools/tests/utils/executeinparallel.js

```diff
@@ -334,6 +334,56 @@ describe( 'executeInParallel()', () => {
 		await promise;
 	} );
 
+	it( 'should use number of cores divided by two as default (`concurrency`)', async () => {
+		const promise = executeInParallel( defaultOptions );
+		await delay( 0 );
+
+		expect( stubs.WorkerMock.instances ).toHaveLength( 2 );
+
+		// Workers did not emit an error.
+		for ( const worker of stubs.WorkerMock.instances ) {
+			getExitCallback( worker )( 0 );
+		}
+
+		await promise;
+	} );
+
+	it( 'should round down to the closest integer (`concurrency`)', async () => {
+		const options = Object.assign( {}, defaultOptions, {
+			concurrency: 3.5
```

Comment:
We need to duplicate this test by calling the `executeInParallel` function with the default value of `concurrency` (which is mocked: `os().cpus().length` returns `7`)

Reply:
I think if we test `concurrency: 3.5`, there is no need to test the behaviour for default value, because it behaves exactly the same as the value passed by us.

I think we just need to check if the default value is actually used.

This simplifies the tests, because it's hard to mock consts.

Reply:
The original issue was reported by using `os.cpus()` construction and I want to make sure (even if it duplicated a case when passing `concurrency` manually) it works as expected.